### PR TITLE
Enable revive

### PR DESCRIPTION
**Pull request by vishwas-sharma2480**
## Description

This pull request addresses issue #1541.

### Changes Made
- Enabled the revive linter rule.
- Resolved all linter errors to ensure the tests pass.

### CODE REVIEW

**Comment by vishwas-sharma2480** [comment]
@BupycHuk  and @JiriCtvrtka  can you please review this PR I have resolved all the remaining linter warnings

File: managed/services/backup/pitr_timerange_service.go [hunk 1]
```diff
@@ -181,7 +181,7 @@ func (s *PBMPITRService) ListPITRTimeranges(ctx context.Context, storage Storage
 // trimTimelines adds one second to the Start value of every timeline record. Required to fit PBM values.
 func trimTimelines(timelines []Timeline) {
 	for i := range timelines {
-		timelines[i].Start += 1
+		timelines[i].Start += u1 //nolint:revive
```
Comment (BupycHuk) [inline managed/services/backup/pitr_timerange_service.go:184]: this change generates error below
```
Error: 24 Jan 24 23:32 UTC ERR Error parsing file error="-: # github.com/percona/pmm/managed/services/backup\n./pitr_timerange_service.go:184:25: undefined: u1" dry-run=false version=v2.39.1
```

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: managed/services/backup/pitr_timerange_service.go (hunk 1 above)
Comment (vishwas-sharma2480) [reply managed/services/backup/pitr_timerange_service.go]: I have made the requested changes please let me know if any further changes are required

**Comment by JiriCtvrtka** [comment]
@vishwas-sharma2480 Hi, could you fix license headers to pass last main check? Thank you.

**Comment by vishwas-sharma2480** [comment]
Hi @JiriCtvrtka I cleared the error it showed for main can you please review the changes

**Review by JiriCtvrtka — APPROVED** [review]

File: managed/services/agents/versioner.go [hunk 2]
```diff
@@ -118,7 +118,7 @@ type MongoDB struct{}
 
 func (*MongoDB) isSoftware() {}
 
-// Name returns the software name for MongoDB.{}
+// Name returns the software name for MongoDB.{}.
```
Comment (ademidoff) [inline managed/services/agents/versioner.go:121]:
```suggestion
// Name returns the software name for MongoDB.
```

**Review by ademidoff — APPROVED** [review]

**Comment by JiriCtvrtka** [comment]
@vishwas-sharma2480 I did some changes to pass main check and PR should be merged soon. Thank you a lot for your contribution.

**Comment by JiriCtvrtka** [comment]
@BupycHuk PTAL

**Comment by vishwas-sharma2480** [comment]
hi @JiriCtvrtka  are there any mo0re chyanges required i see the issue is still open ?

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**